transformations/butter_fingers_perturbation/test.json

```diff
@@ -2,26 +2,32 @@
   "type": "butter_fingers_perturbation",
   "test_cases": [
     {
+      "class": "ButterFingersPerturbation",
```

Comment:
It's better that this be at the outside level. We will need to repeat this in every test case then.

Reply:
Yeah, I was also thinking. Will do the required changes.